.eslintrc.json

```diff
@@ -2,7 +2,8 @@
     "extends": "airbnb",
     "parser": "babel-eslint",
     "env": {
-        "es6": true
+        "es6": true,
+        "jest": true
```

Comment:
👍
I think it should be removed from here, and added at the discretion of the test files. Either by adding a .eslintrc.json inside the `test` folder, or simply by adding 
```js
/* eslint env jest */
``` 
at the beginning of the test file.